candle-nn/tests/loss.rs

```diff
@@ -39,3 +39,38 @@ fn nll_and_cross_entropy() -> Result<()> {
     assert_eq!(to_vec0_round(&loss, 4)?, 1.1312);
     Ok(())
 }
+
```

Comment:
Could you put in as a comment the equivalent PyTorch snippet that can be used to check the test final output?